guide/inheritance-wallet/wallet-creation.md

```diff
@@ -0,0 +1,338 @@
+   width = 800
+   height = 500
+%}
+
+## Step 1: Wallet configuration
+
+The first is the wallet configuration, where Bob and Alice define the rules for how the wallet should work. In the second part, they add all the signing keys.
+
+#### Creating the primary key set
+
+Bob starts the wallet creation process on his phone, while Alice will use the app on her own phone to add her signing key. The app gives them a choice between two of the most commonly used wallet types: The first one is a standard, single-key wallet. The second option is a 2-of-3 multi-key wallet, which is suited for larger savings. There is also a third option that allows users to create their own custom setups. Bob and Alice choose the 2-of-3 multi-key wallet.
+
+{% include image-gallery.html pages = page.images_wallet-configuration %}
+
+#### Enabling the recovery path
+
+After configuring the primary key set, the app asks the users whether they would like to enable a recovery path. The recovery path uses the same keys that are part of the primary key set and unlocks after 6 months of inactivity. If it is unlocked, only one signature will be required to spend bitcoin, instead of two. We have covered this in the time-based reference design as well.
+
+{% include image-gallery.html pages = page.images_recovery-path %}
+
+#### Creating the inheritance key set
+
+After enabling the recovery path, the app offers users to create a dedicated inheritance key set. The process of configuring the key set is almost the same as for the primary key set.
+
+The main difference is that the inheritance key set should only be unlocked after a certain amount of time. Therefore, Bob is prompted to define the rules under which the key set should be unlocked, after he has chosen the appropriate key set type.
+
+{% include image-gallery.html pages = page.images_inheritance-key-set %}
+
+## Step 2: Adding the signing keys
+
+The wallet is now fully configured. What is left to do is to add the signing keys. Adding a key technically means that users have to import the extended public key (XPUB) from each signing device that should be used to sign transactions. 
+
+### Adding the primary keys
+
+This sounds more complicated than it actually is. Depending on the specific signing devices that are being used, it can be as simple as scanning a QR code, connecting it via NFC or Bluetooth, or by using a USB cable. 
+
+##### Bob adds the first two keys
+Bob gets started and imports the first two keys by himself. One from his personal Trezor and one from a Coldcard that he shares with Alice.
+
+{% include image-gallery.html pages = page.images_primary-keys-bob %}
+
+##### Bob requests the third key from Alice
+Next, it's time to add Alice's key, which will be imported from her Bitbox. Our app makes this process easy, because Bob can just create a key request that Alice can scan with her phone. Alternatively, he can also send it to her via a secure channel.
```

Comment:
```suggestion
Next, it's time to add Alice's key, which will be imported from her BitBox. Our app makes this process easy, because Bob can just create a key request that Alice can scan with her phone. Alternatively, he can also send it to her via a secure channel.
```